src/shared/models.ts

```diff
@@ -15,3 +15,16 @@ export enum ModelAppStatus {
   Error = 'Error',
   Unregistered = 'Unregistered',
 }
+
+export type ModelInfo = {
```

Comment:
there is a lot of duplication between `ModelAppConfig` defined in config.ts and `ModelInfo`. We should only put fields in one or the other. Perhaps ModelInfo here can exclude the first 5 fields.